cypress/e2e/settings/hidden-rr-migration.spec.ts

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright 2022 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/// <reference types="cypress" />
+
+import { SynapseInstance } from "../../plugins/synapsedocker";
+
+function seedLabs(synapse: SynapseInstance, labsVal: boolean | null) {
```

Comment:
Return type

Reply:
I don't think we require a return type if it's `void`?

Reply:
I don't think so? https://github.com/vector-im/element-meta/blob/travis/temp-code-style-web/web-code-style.md?plain=1#L193 What am I missing?

Reply:
> Notable exceptions are [...], when a void return type is obvious, [...]

😇 

we should probably add them though, I guess

Reply:
> we should probably add them though, I guess

I would rather we we do type returns :p But let's ignore it on this PR :)